atlite/convert.py

```diff
@@ -62,10 +62,11 @@ def convert_and_aggregate(
 
     Parameters
     -----------
-    matrix : N x S - xr.DataArray or sp.sparse.csr_matrix or None
+    matrix : N x S or N x X x Y- xr.DataArray or sp.sparse.csr_matrix or None
```

Comment:
TBH I don't like it.

`matrix` by itself is already a slightly confusing construct. Overloading it now to be able to accept a different datatype with a different dimension doesn't help with the confusion, but increases it.

Can we have a dedicated argument for `xr.DataArrays`? (I like the idea but not the implementation as an overloaded function parameter)